component/core/src/main/java/org/exoplatform/social/core/space/SpaceUtils.java

```diff
@@ -1937,10 +1937,9 @@ public static String getIconClass(UserNode node) {
       return null;
     }
     if (isHomeNavigation(node)) {
-      return "uiIconAppSpaceHomePage uiIconDefaultApp";
+      return "fa-stream";
     }
-    String appName = node.getPageRef().getName();
-    return "uiIconApp" + node.getName() + " uiIconApp" + appName + " uiIconDefaultApp";
+    return "fa-icon-" + node.getName();
```

Comment:
No need to make it variablized, but you can simply make a computed value to get the icon to use for each page or, you can also inherit CSS class in less instead of copying the properties value.

Reply:
done